build/release.js

```diff
@@ -272,11 +280,28 @@ function writeJsonSync( fname, json ) {
 	}
 }
 
-function copy( oldFile, newFile ) {
-	status( "Copying " + oldFile + " to " + newFile );
-	if ( !dryrun ) {
-		fs.writeFileSync( newFile, fs.readFileSync( oldFile, "utf8" ) );
+function fixMinRef( oldText ) {
+	var mapRef = new RegExp( "^//# sourceMappingURL=jquery-migrate.min.map\\n?", "m" );
+
+	// Remove the ref for now rather than try to fix it
+	var newText = oldText.replace( mapRef, "" );
+	if ( oldText === newText ) {
+		throw "fixMinRef: Unable to find the sourceMappingURL";
+	}
+	return newText;
+}
+
+function fixMapRef( oldText, newFile ) {
+	var mapJSON = JSON.parse( oldText );
+	var sources = mapJSON.sources;
+	if ( sources.join() !== "../src/migratemute.js,jquery-migrate.js" ) {
+		throw "fixMapRef: Unexpected sources entry: " + sources;
 	}
+
+	// This file isn't published, not sure the best way to deal with that
+	sources[ 0 ] = "migratemute.js";
```

Comment:
Why do we build it if we don't publish it?

Reply:
Right now the way we turn off warnings in the minified file is to [include `migratemute.js` in front of the standard file](https://github.com/jquery/jquery-migrate/blob/master/Gruntfile.js#L79-L80) and run uglify on that. As a result it makes a map that mentions both files. However we only publish the standard unminified file, not `migratemute.js`. I don't see any direct uglify options to deal with this kind of thing, which isn't surprising TBH because it's a pretty strange way to make a map. I've got some ideas on how to improve it but am thinking it might be best to just publish first since this map isn't even referenced in the minified file at the moment.